core-bundle/src/Resources/contao/library/Contao/Config.php

```diff
@@ -373,8 +373,8 @@ public static function get($strKey)
 	/**
 	 * Temporarily set a configuration value
 	 *
-	 * @param string $strKey   The short key
-	 * @param string $varValue The configuration value
+	 * @param string     $strKey   The short key
+	 * @param mixed|null $varValue The configuration value
```

Comment:
Isn't `null` included in `mixed`?

Reply:
Yes it is, but the legacy code uses this notations all around. :slightly_smiling_face:  Shall I strip it for both occurrences in this class?

Reply:
Nevermind, I stripped it for both occurrences in a07ffbfb42c1d42f3f325c8be33e5c99a7bd6e04.